pkg/provider/flipt/provider.go

```diff
@@ -225,25 +186,9 @@ func (p Provider) BooleanEvaluation(ctx context.Context, flag string, defaultVal
 
 // StringEvaluation returns a string flag.
 func (p Provider) StringEvaluation(ctx context.Context, flag string, defaultValue string, evalCtx of.FlattenedContext) of.StringResolutionDetail {
-	// TODO: we have to check if the flag is enabled here until https://github.com/flipt-io/flipt/issues/1060 is resolved
-	f, res, err := p.getFlag(ctx, flag)
-	if err != nil {
-		return of.StringResolutionDetail{
-			Value:                    defaultValue,
-			ProviderResolutionDetail: res,
-		}
-	}
-
-	if !f.Enabled {
-		return of.StringResolutionDetail{
-			Value: defaultValue,
-			ProviderResolutionDetail: of.ProviderResolutionDetail{
-				Reason: of.DisabledReason,
-			},
-		}
-	}
+	namespace, flagKey := splitNamespaceAndFlag(flag)
```

Comment:
i think we should look and see if there is an alternative way of supporting `namespace` without having to rely on callers pretending it to the `flag` string here.

I don't think the OpenFeature `flag` string is meant to be used this way, so it may be better to set the necessary `namespace` when constructing the provider (via a functional option). Then if the namespace is not set, it will use the `default` namespace.

Reply:
ie:

```
provider := flipt.NewProvider(flipt.ForNamespace("production"))
```

We are limited by the contract of the open feature provider, but it seems cleaner to me to try and stick with that contract if possible without adding our own meanings on top (`flag` is flagKey, not `namespace/flagKey`)

Reply:
@markphelps Yeah I thought about this as well. So it seems as though we can only support one namespace per provider. That was the reason I chose to do it in this way. But I am for whatever makes sense.